### PR TITLE
Adding ATLAS autoencoders compression project

### PR DESCRIPTION
**Pull request by urania277**

### CODE REVIEW

**Comment by aflp91** [comment]
Caterina, you have to remove the first empty column in your proposal (ill format)
I think you should rename your proposal with an underscore.
And, please, you need to emphasize the main mentor.

**Comment by urania277** [comment]
Thanks @aflp91! I am not sure what you mean with the first empty column in the proposal (apologies if it's obvious...), I should have done the rest.

File: _gsocproposals/2021/proposal_ATLASCompressionAE.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+ Jupyter and pyTorch exercise of running the existing NN and describing the results. 
+
+ ## Links
+  * [ROOT](https://root.cern/)
+  * [Jupyter](http://jupyter.org)
+  * [PyTorch](http://pytorch.org)
+
+ ## Mentors
+  * [Caterina Doglioni](mailto:caterina.doglioni@cern.ch) [main]
```
Comment (xvallspl) [inline _gsocproposals/2021/proposal_ATLASCompressionAE.md:54]: The main mentor should be in bold letters

File: _gsocproposals/2021/proposal-ATLASCompressionAE.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ An improved autoencoder network with documentation and figures of merit that may
   * [PyTorch](http://pytorch.org)
 
  ## Mentors
-  * [Caterina Doglioni](mailto:caterina.doglioni@cern.ch)
+  * [Caterina Doglioni](mailto:caterina.doglioni@cern.ch) [main]
```
Comment (aflp91) [inline _gsocproposals/2021/proposal-ATLASCompressionAE.md]: Instructions are to put the main mentor in bold letters ( * **[Caterina Doglioni](mailto:caterina.doglioni@cern.ch)**)

**Review by xvallspl — CHANGES_REQUESTED** [review]
Antoine was referring to the document sections being indented one space in.

File: _gsocproposals/2021/proposal_ATLASCompressionAE.md (hunk 1 above)
Comment (urania277) [reply _gsocproposals/2021/proposal_ATLASCompressionAE.md]: thanks, should be fixed

**Review by aflp91 — APPROVED** [review]
Looks OK with me.

**Comment by agheata** [comment]
All comments addressed. Dismissing un-closed reviews just to speed things up and merge.

**Review by xvallspl — APPROVED** [review]

**Pull request merged**